k-distribution/include/kframework/builtin/domains.md

```diff
@@ -550,7 +550,7 @@ elements present in either set.
 
 ```k
   syntax Set ::= Set "|Set" Set            [left, function, functional, hook(SET.union)]
-  rule S1:Set |Set S2:Set => S1 (S2 -Set S1)
+  // FIXME rule S1:Set |Set S2:Set => S1 (S2 -Set S1)
```

Comment:
I removed this rule to make the added `|Set` simplification rules kick in. A few `union` tests would not produce the desired output otherwise. OK to delete it?

Reply:
If you're removing this you need to add an implementation of this hook to the Haskell backend.